src/main/java/it/pagopa/pagopa/apiconfig/controller/HomeController.java

```diff
@@ -55,10 +60,12 @@ public RedirectView home() {
 
     @GetMapping("/info")
     public ResponseEntity<AppInfo> healthCheck() {
```

Comment:
couldn't we do it with [actuator](https://www.baeldung.com/spring-boot-health-indicators#3-additional-details) ?

Reply:
as said in other meeting, to adopt actuator properly, we should define who and how will consume info.
for time restriction of the project we could assume this way is preferable to reach the goal.